### PR TITLE
 Handle error responses without partial error info.

### PR DESCRIPTION
**Pull request by igorpeshansky**

### CODE REVIEW

File: test/plugin/constants.rb [hunk 1]
```diff
@@ -13,6 +13,33 @@
 # limitations under the License.
 
 # Constants used by unit tests for Google Cloud Logging plugin.
+module Google
+  module Protobuf
+    Any.class_eval do
+      def self.pack(msg, type_url_prefix = 'type.googleapis.com/')
+        any = Google::Protobuf::Any.new
+        any.pack(msg, type_url_prefix)
```
Comment (igorpeshansky) [inline test/plugin/constants.rb]: This was google/protobuf#4719, which just got merged.

Comment (igorpeshansky) [inline test/plugin/constants.rb]: This was google/protobuf#4719, which just got merged.

Comment (qingling128) [reply test/plugin/constants.rb]: Can we add a TODO to clean this up once we upgrade to the newer protobuf version?

Comment (qingling128) [reply test/plugin/constants.rb]: Friendly ping.

**Pull request merged**